### PR TITLE
Fixes some runtimes

### PR DESCRIPTION
**Pull request by Shadowlight213**
Fixes runtimes in Dna console, message monitor, and igniter

### CODE REVIEW

File: code/game/machinery/computer/message.dm [hunk 1]
```diff
@@ -269,16 +269,19 @@
 	if ((usr.contents.Find(src) || (in_range(src, usr) && istype(src.loc, /turf))) || (istype(usr, /mob/living/silicon)))
 		//Authenticate
 		if (href_list["auth"])
-			if(auth)
-				auth = 0
-				screen = 0
+			if(src.linkedServer == null || (src.linkedServer.stat & (NOPOWER|BROKEN)))
```
Comment (duncathan) [inline code/game/machinery/computer/message.dm]: Can just be `if(!linkedServer || linkedServer.stat & (NOPWER|BROKEN))`

File: code/modules/assembly/igniter.dm [hunk 2]
```diff
@@ -22,7 +22,7 @@
 	if(!..())	return 0//Cooldown check
 	var/turf/location = get_turf(loc)
 	if(location)	location.hotspot_expose(1000,1000)
-	sparks.start()
+	if(sparks)	sparks.start()
```
Comment (duncathan) [inline code/modules/assembly/igniter.dm]: Why would sparks be null? If that shouldn't be happening then this is a bandaid and not an actual solution

Comment (Shadowlight213) [inline code/modules/assembly/igniter.dm]: I'm not sure to be honest. I'll remove this part then.

**Comment by Shadowlight213** [comment]
@Jordie0608 @Aranclanos @Razharas @Miauw62 @RemieRichards

**Pull request merged**